src/EA.Weee.Web/Areas/Admin/Views/Scheme/Overview/RegisteredCompanyDetailsOverview.cshtml

```diff
@@ -25,7 +25,7 @@
                         <td class="govuk-table__cell">@Model.CompanyName</td>
                     </tr>
                     <tr class="govuk-table__row">
-                        <th class="govuk-table__header" scope="row">Company registration number (CRN)</th>
+                        <th class="govuk-table__header" scope="row">company registration number (CRN)</th>
```

Comment:
Dont think this should change as its heading.

Reply:
restored